libs/sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -548,6 +548,8 @@ definedDataType
     : DOUBLE PRECISION
     | TIMESTAMP WITHOUT TIME ZONE
     | TIMESTAMP WITH TIME ZONE
+    | TIME WITHOUT TIME ZONE
+    | TIME
```

Comment:
it can be removed, the defined data type rule is used to define types that cannot be matched with ident, e.g. data types with whitespaces.

Reply:
remove `time` ?